### ISSUE TITLE
Feature Request: Subscribing to Twitter Hashtag

### ISSUE THREAD

**Issue by AndyHee** [friendica/friendica#8361]
It would be great if saved hashtags could also (optionally) pull in posts from the Twitter network.

There is no Twitter login required for this. Here a random URL: https://twitter.com/hashtag/strike?src=hashtag_click

**Comment by MrPetovan** [comment]
There's a bit of scope definition to be done here. The connector scope always has been around your contacts and your timeline. This provides a limited amount of data because there's a limited amount of posts your own contacts can write. However, if we start importing the results from a public Twitter search, all bets are off, because beyond the search results firehose, we also import the whole post ascendency in the threads until the top-level post.

By comparison, the current saved search follow in Friendica is based on the public posts the node is aware of, and it doesn't "pull" anything. The relay feature with user tags is the closest feature I can think of, but the potential amount of imported posts still is pretty low compared to Twitter's and it works with push as well. Twitter doesn't publish XML feeds for search either.

So this would be an entirely new feature, not a hook, and it should be well-defined, including in terms of frequency of pulls and imported post number limit.

**Comment by tobiasd** [comment]
Is there maybe a 3rd party providing RSS Feeds for search results on Twitter? You could follow these RSS contacts then.

**Comment by AndyHee** [comment]
> So this would be an entirely new feature, not a hook, and it should be well-defined, including in terms of frequency of pulls and imported post number limit.

Thanks @MrPetovan for the detailed reply. Yes, that's what I initially thought too that linking this to existing relay search terms is not feasible nor desirable. So we're possibly looking for an entirely new addon. This may potentially have a limited number of search terms (e.g. 5) one save and may cap of many replies it can pull in (e.g. 100). It would be away to reduce the risk of grinding a node to hold by users adding many search terms.

Sure, Twitter not publishing XML feeds it this this more difficult. What's is the situation with  searching and pulling content via API; is this more realistic? 


> Is there maybe a 3rd party providing RSS Feeds for search results on Twitter? You could follow these RSS contacts then.

Thanks @tobiasd for this! I've just looked into this. The options very limited most feed generators don't work (well) or are only commercial offers. Self-hosting scripts don't exist or are hopelessly out-of-date.  However, I managed to find one provider https://rss.app that seems to work so far  (pulling formatted text of tweets). I've reduced the rate Friendica pulls the rss feed and hope they won't block me. Definitely a quick workaround, if it works. Thanks.
